alvr/common/src/data/session.rs

```diff
@@ -112,6 +112,7 @@ pub struct ClientConnectionDesc {
 #[serde(rename_all = "camelCase")]
 pub struct SessionDesc {
     pub setup_wizard: bool,
+    pub i18n_locale: String,
```

Comment:
I would like to keep names not linked to specific libraries. "locale" should be enough.

Reply:
Naming changed